Modules/Relationships/src/Relationships.ConsumerApi/Controllers/RelationshipsController.cs

```diff
@@ -165,9 +146,6 @@ public async Task<IActionResult> RevokeRelationshipReactivation([FromRoute] stri
     [ProducesError(StatusCodes.Status404NotFound)]
     public async Task<IActionResult> AcceptReactivationOfRelationship([FromRoute] string id, CancellationToken cancellationToken)
     {
-        var peerOfActiveIdentityInRelationshipResponse = await _mediator.Send(new GetPeerOfActiveIdentityInRelationshipQuery { Id = id }, cancellationToken);
-        await EnsurePeerIsNotToBeDeleted(peerOfActiveIdentityInRelationshipResponse.IdentityAddress, cancellationToken);
-
```

Comment:
Same as with accepting relationship while to be deleted.

Reply:
Waiting for Timo's answer.